rules/windows/defense_evasion_clearing_windows_security_logs.toml

```diff
@@ -0,0 +1,36 @@
+[metadata]
+creation_date = "2020/11/12"
+maturity = "production"
+updated_date = "2020/11/12"
+
+[rule]
+author = ["Anabella Cristaldi"]
+description = """
+Identifies attempts to clear Windows event log stores. This is often done by attackers in an attempt to evade
+detection or destroy forensic evidence on a system.
+"""
+from = "now-9m"
+index = ["winlogbeat-*"]
```

Comment:
```suggestion
index = ["winlogbeat-*", "logs-endpoint.events.*"]
```

Reply:
I think `event.action:("audit-log-cleared" or "Log clear")` appears in the Windows event logs that are shipped by Winlogbeat, but not in our endpoint logs.